NumberValidator/Validators/IN/EPIC.cs

```diff
@@ -0,0 +1,54 @@
+﻿/*EPIC (Electoral Photo Identity Card, Indian Voter ID).
+
+The Electoral Photo Identity Card (EPIC) is an identity document issued by
+the Election Commission of India (ECI) only to the India citizens who have
+reached the age of 18.
+
+Each EPIC contains an unique 10 digit alphanumeric identifier known as EPIC
+number or Voter ID number.
+
+Every EPIC number begins with a Functional Unique Serial Number (FUSN), a 3
+letter unique identifier for each Assembly Constituency. FUSN is followed by
+a 6 digit serial number and 1 check digit of the serial number calculated
+using Luhn algorithm.*/
+
+using System.Text.RegularExpressions;
+using NumberValidator.Helpers;
+
+namespace NumberValidator.Validators.IN
+{
+    public class EPIC : IValidator
+    {
+        private static readonly Regex _EPIC_RE = new Regex(@"^[A-Z]{3}[0-9]{7}$", RegexOptions.Compiled);
```

Comment:
this snake case naming is not correct. since it is a variable keep it as camel case.

Reply:
corrected